application/models/Logbook_model.php

```diff
@@ -4418,7 +4431,7 @@ function wpx($testcall, $i) {
       $c = '';
 
       $lidadditions = '/^QRP$|^LGT$/';
-      $csadditions = '/^T$|^P$|^R$|^A$|^M$|^LH$/';
+      $csadditions = '/^X$^D$^T$|^P$|^R$|^A$|^M$|^LH$/';
```

Comment:
am i wrong, or are we missing an OR `|` here at the RegEx?

Reply:
No, you are absolutely right. Thanks for spotting!